examples/nvidia-vgpu-device-manager-mig-support-example.yaml

```diff
@@ -0,0 +1,119 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: vgpu-device-manager
+  namespace: default
+  labels:
+    app: vgpu-device-manager
+spec:
+  selector:
+    matchLabels:
+      app: vgpu-device-manager
+  template:
+    metadata:
+      labels:
+        app: vgpu-device-manager
+    spec:
+      serviceAccount: vgpu-device-manager
+      serviceAccountName: vgpu-device-manager
+      containers:
+      - name: vgpu-device-manager
+        image: nvcr.io/nvidia/cloud-native/vgpu-device-manager:v0.1.0
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: NAMESPACE
+          value: "gpu-operator"
```

Comment:
Question: Since this DaemonSet is configured to run in the `default` namespace, should this also be `default`?

Reply:
I am fine with a separate PR to update the samples.